x/upgrade/autocli.go

```diff
@@ -50,14 +50,17 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 			Service: upgradev1beta1.Msg_ServiceDesc.ServiceName,
 			RpcCommandOptions: []*autocliv1.RpcCommandOptions{
 				{
-					RpcMethod: "SoftwareUpgrade",
-					Skip:      true, // skipped because authority gated
+					RpcMethod:   "CancelUpgrade",
+					Use:         "cancel-upgrade-proposal",
+					Short:       "Submit a proposal to cancel a planned chain upgrade.",
+					GovProposal: true,
 				},
 				{
-					RpcMethod: "CancelUpgrade",
+					RpcMethod: "SoftwareUpgrade",
```

Comment:
We don't need this one?

Reply:
it was actually just moved up, diff is weird